### PR TITLE
Merge240

### PR DESCRIPTION
**Pull request by kdlong**
Make 240 master branch. This should be a trivial merge, with changes from the mg240 branch only affecting scripts, patches and some example cards. I've tested with a couple of processes (one with reweight+pre-compilation) and don't see issues, but a double check would be nice.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat [hunk 1]
```diff
@@ -43,104 +50,113 @@ average = event_norm ! Normalize events to sum or average to the X sect.
...
+  1.0  = epsgamma  ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ True  = isoEM  ! isolate photons from EM energy (photons and leptons)
 #***********************************************************************
-# Maximal PDG code for quark to be considered a jet when applying cuts.*
-# At least all massless quarks of the model should be included here.   *
+# For aMCfast+APPLGRID use in PDF fitting (http://amcfast.hepforge.org)*
 #***********************************************************************
- 5 = maxjetflavor
+ 0 = iappl ! aMCfast switch (0=OFF, 1=prepare grids, 2=fill grids)
 #***********************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat]: has this parameter disappeared in 240 ?

**Comment by bendavid** [comment]
It gets set automatically now based on the jet definition in the process card.

**Comment by perrozzi** [comment]
shall we simply merge?

**Comment by bendavid** [comment]
See no reason not to.

**Pull request merged**